docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,162 @@
+Run the workflow using custom data
+==================================
+
+In this tutorial, you will run the workflow using custom focal data in addition to the example reference data. The reference data will serve as background context for the new data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`example-data`. These instructions will set up the command line environment used in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
+
+We will retrieve 10 sequences from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__ and select **Login**.
+
+   .. image:: ../images/gisaid-homepage.png
+      :width: 400
+      :alt: GISAID login link
+
+2. Login to your GISAID account.
+
+   .. image:: ../images/gisaid-login.png
+      :width: 200
+      :alt: GISAID login
+
+3. In the top left navigation bar, select **EpiCoV** then **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+4. Select the first 10 records.
+
+   .. note::
+
+      You can filter and select a different set of records here, but note that more records will take the workflow longer to run, and the GISAID web interface has a maximum of 5,000 records per download.
+
+   .. image:: ../images/gisaid-select-records.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+5. Select **Download** in the bottom right of the search results.
+6. Select **Input for the Augur pipeline** as the download format.
+7. Select **Download**.
+
+   .. image:: ../images/gisaid-augur-pipeline-download.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You may see different download options, but it is fine as long as **Input for the Augur pipeline** is available.
+
+8. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+9.  Move these extracted files to the ``data/`` directory in the ``ncov/`` directory.
+10. Rename the files as ``custom.metadata.tsv`` and ``custom.sequences.fasta``.
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a pre-written ``--configfile``:
+
+.. code:: text
+
+   nextstrain build . --cores 4 --configfile ncov-tutorial/custom-data.yaml
```

Comment:
This ran as expected (in 6 min with 3 CPUs), but the workflow printed a bunch of annoying warnings about the lack of a subsampling scheme for the build like this one:

```
WARNING: No valid subsampling scheme is defined for build 'custom-build'. Skipping subsampling and using all available samples.
```

This is a warning that I added to the workflow back when I changed the default behavior to include all samples (last spring or summer). We could either drop the warning completely or update the builds entry to explicitly specify the subsampling scheme like this:

```
builds:
  custom-build:
    subsampling_scheme: all
```

with the explanation that we will define a better scheme in the next part of the tutorial.

I sort of lean toward removing the warning, but maybe there is pedagogical value in exposing the subsampling scheme logic earlier?

Reply:
After sleeping on this, I've changed my mind and slightly prefer exposing the `subsampling_scheme: all` in this "custom data" tutorial. This will preemptively answer the question that users would probably ask themselves about "which data are going into this analysis?". We can say something like "For now, we will set a subsampling scheme that uses all input data; in the next section of the tutorial, we will show how to customize this scheme for genomic surveillance".

Reply:
Is `all` a predefined subsampling scheme? I think also good to add this to [the configuration guide](https://docs.nextstrain.org/projects/ncov/en/latest/reference/configuration.html).

Reply:
Yeah, [it's one of the few predefined schemes were provide](https://github.com/nextstrain/ncov/blob/a5a7384ff2783b5f59677bb9c50f02a1c3c04feb/defaults/parameters.yaml#L200-L204).

Reply:
Fixed with 03e713ea, https://github.com/nextstrain/ncov-tutorial/commit/aef9f19.

Created a separate issue to document predefined subsampling schemes: #900.

Reply:
Will link to docs on predefined subsampling schemes once #905 is merged.